src/views/studio/EditVideoSheet/EditVideoSheet.tsx

```diff
@@ -66,6 +69,22 @@ export const EditVideoSheet: React.FC = () => {
       channelId_eq: activeChannelId,
     },
   })
+  const { DataLostWarningDialog, openWarningDialog } = useDisplayDataLostWarning()
+
+  useEffect(() => {
+    if (sheetState === 'closed' || !haveVideoTabsAssetCache) {
+      return
+    }
+
+    const beforeUnload = (e: BeforeUnloadEvent) => {
+      e.preventDefault()
+      e.returnValue = 'Do you want to leave this page? Changes that you made may not be saved.'
```

Comment:
Is this actually respected in any way? Seems all the dialog have default values?

Reply:
You're right, I think it's deprecated